packages/blockchain/tron/src/lib/tron.sdk.ts

```diff
@@ -0,0 +1,35 @@
+import { BlockchainTronService } from '@tatumio/api-client'
+import { Blockchain } from '@tatumio/shared-core'
+import { SDKArguments } from '@tatumio/shared-abstract-sdk'
+import { abstractBlockchainSdk } from '@tatumio/shared-blockchain-abstract'
+import { tronWeb } from './services/tron.web'
+import { tronTx } from './services/tron.tx'
+import { tronWallet } from './services/tron.wallet'
+import { tronRecord } from './services/tron.record'
+
+const blockchain = Blockchain.TRON
+
+export const TatumTronSDK = (args: SDKArguments) => {
+  const web = tronWeb()
+  const api = BlockchainTronService
+
+  return {
+    ...abstractBlockchainSdk({
+      ...args,
+      blockchain,
+    }),
+    transaction: tronTx({ tronWeb: web }),
+    api,
```

Comment:
please remove api

Reply:
Ok, but there is api for evm-based chains. Why not for tron?

Reply:
Actually I want to get rid of API field from everywhere. we already have those methods in blockchain

Reply:
Ok, got it.